README.md

```diff
@@ -149,12 +149,12 @@ Code checks:
 
 ```bibtex
 @misc{olmo20242olmo2furious,
-      title={2 OLMo 2 Furious}, 
-      author={Team OLMo and Pete Walsh and Luca Soldaini and Dirk Groeneveld and Kyle Lo and Shane Arora and Akshita Bhagia and Yuling Gu and Shengyi Huang and Matt Jordan and Nathan Lambert and Dustin Schwenk and Oyvind Tafjord and Taira Anderson and David Atkinson and Faeze Brahman and Christopher Clark and Pradeep Dasigi and Nouha Dziri and Michal Guerquin and Hamish Ivison and Pang Wei Koh and Jiacheng Liu and Saumya Malik and William Merrill and Lester James V. Miranda and Jacob Morrison and Tyler Murray and Crystal Nam and Valentina Pyatkin and Aman Rangapur and Michael Schmitz and Sam Skjonsberg and David Wadden and Christopher Wilhelm and Michael Wilson and Luke Zettlemoyer and Ali Farhadi and Noah A. Smith and Hannaneh Hajishirzi},
+      title={{2 OLMo 2 Furious}},
```

Comment:
Is this needed?

Reply:
capitalization in our title keeps getting mangled up in citations because of single curly braces. Can't fix what Google Scholar does, but at least we can prevent that here!

<img width="365" alt="Screenshot 2025-03-25 at 15 01 29" src="https://github.com/user-attachments/assets/24d6cb59-6f75-47bc-a539-9733e26637fc" />